ocaml/xapi/xapi_session.ml

```diff
@@ -303,8 +303,9 @@ let do_local_change_password uname newpwd =
 let trackid session_id = Context.trackid_of_session (Some session_id)
 
 (* finds the intersection between group_membership_closure and pool's table of subject_ids *)
-let get_intersection ~__context subject_ids_in_db subject_identifier
-    group_membership_closure =
+let get_intersection ~(__context : Context.t) subject_ids_in_db
```

Comment:
Why add the type here?

Reply:
It's a leftover that I forgot to remove. Thanks!